CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 ### Changed
 - Most built-in logs like `Finished an outgoing client HTTP request` now use the context logger. 
+- **Context Extractor**: Added [`ContextExtractor`](https://godoc.org/github.com/uber/zanzibar/runtime#ContextExtractor) interface. It can be used to define "extractors" or functions to pull out dynamic fields like trace ID, request headers, etc. out of the context to be used in log fields and metric tags. These can be used to pull out fields that are application-specific without adding code to zanzibar.
+- **PackageRoot**: Added PackageRoot for definition of root folder
```

Comment:
You don't have to document this. The changelog isn't for exhaustively documenting every minor change. It's for notable changes. If you want to put it in the changelog you should be describing what it is, why one would use it, etc. but it is probably too minor to include here.